RCTMapboxGL/RCTMapboxGLManager.m

```diff
@@ -186,9 +186,31 @@ - (NSDictionary *)customDirectEventTypes
                             
                             RCTMGLAnnotation *pin = [[RCTMGLAnnotation alloc] initWithLocationRightCallout:CLLocationCoordinate2DMake(coordinate.latitude, coordinate.longitude) title:title subtitle:subtitle id:id rightCalloutAccessory:imageButton];
                             [pins addObject:pin];
+                            
+                            if ([anObject objectForKey:@"annotationImage"]) {
+                                NSObject *annotationImage = [anObject valueForKey:@"annotationImage"];
+                                NSString *annotationImageURL = [annotationImage valueForKey:@"url"];
+                                CGFloat height = (CGFloat)[[annotationImage valueForKey:@"height"] floatValue];
+                                CGFloat width = (CGFloat)[[annotationImage valueForKey:@"width"] floatValue];
+                                CGSize annotationImageSize =  CGSizeMake(height, width);
+                                pin.annotationImageURL = annotationImageURL;
+                                pin.annotationImageSize = &(annotationImageSize);
```

Comment:
Once you take care of https://github.com/mapbox/react-native-mapbox-gl/pull/75/files#r35726698, the `&` can go away.